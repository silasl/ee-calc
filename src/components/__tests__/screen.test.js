import React from 'react';
import { shallow } from 'enzyme';

import Screen from '../screen';

describe(Screen, () => {
    it('should render correctly', () => {
        expect(shallow(<Screen />)).toMatchSnapshot();
    });

    it('should render correctly with a value', () => {
        expect(shallow(<Screen value="Foo" />)).toMatchSnapshot();
    });
});
