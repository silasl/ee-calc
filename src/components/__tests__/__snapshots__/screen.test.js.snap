// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Screen should render correctly 1`] = `<styled.div />`;

exports[`Screen should render correctly with a value 1`] = `
<styled.div>
  Foo
</styled.div>
`;
